client/components/domains/example-domain-browser/index.jsx

```diff
@@ -0,0 +1,32 @@
+/** @format */
```

Comment:
Move example browser svg to its own component so it can be reused in the import url step.